packages/amplify-codegen/src/commands/models.js

```diff
@@ -145,6 +146,13 @@ function getModelOutputPath(context) {
   }
 }
 
+function getAmplifyCLIVersionComment(context) {
+  if (context.usageData == null || !context.usageData.version) {
+    return null
+  }
+  return '// Generated using Amplify CLI version: ' + context.usageData.version;
```

Comment:
nit: is it better to define a const and use it, rather than giving the string here directly?

Reply:
I am not sure if `const comment = '// Generated using Amplify CLI version: ' + context.usageData.version; return comment` is any different in this case.

Reply:
What I meant was to declare something like this:
<img width="435" alt="Screen Shot 2021-03-18 at 11 25 50 AM" src="https://user-images.githubusercontent.com/2407186/111677745-be49e200-87dc-11eb-854f-acf5b43e6327.png">

Reply:
This is not exactly a constant string like the `MOCK_GENERATED_CODE`. We are constructing it inside this method and calling it only from a single method (`generateModels`) right now. Adding a global constant might not be a good approach right now.